test/test-cases/regression/rule-920450.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "enabled":1,
+    "version_min":300000,
+    "title":"OWASP CRS id:920450",
```

Comment:
I would prefer to have a title with a more meaningful message. If the test "**OWASP CRS id:920450**" fails, what does it means to the user?

Reply:
Changed in [ceb0e3](https://github.com/SpiderLabs/ModSecurity/pull/2297/commits/ceb0e381d0338442b6d4761c9847630202611673#diff-e641572b2bd6ff4e44759e3dea1d59cfe9475a364258bdfac6c25628efea2bd1R5)